resqpy/well.py

```diff
@@ -220,6 +220,11 @@ def create_xml(self, add_as_part = True,
       self.root_node = datum
 
       return datum
+   
+   def __eq__(self, other):
```

Comment:
Might be worth putting a comment in the code to indicate that this is a temporary implementation of __eq__ as a class like MdDatum is worthy of having a fully implemented version